ubuntu/gpu/rapids/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM databricksruntime/gpu-python:cuda10.1
+FROM databricksruntime/gpu-conda:cuda10.1
```

Comment:
don't understand how this reduces the image size so much

Reply:
There are three major places we saved space in this version:
* NVIDIA cuda container does not install all the GPU libraries we pre-installed, e.g., the full CUDA toolkit and development libraries.
* We did a "conda clean --all" in the base environment.
* The RAPIDS container used to depend on gpu-python, which has a conda env "gpu" already.